lnbits/core/migrations.py

```diff
@@ -301,3 +301,52 @@ async def m010_create_installed_extensions_table(db):
         );
     """
     )
+
+
+async def m011_add_deleted_to_wallets(db):
```

Comment:
i would combine the migrations, into 1

Reply:
I think I would keep it like this, the second one could fail but the first one is very safe. If the column is created but the second stage fails, the migration counter won't be incremented but the column will still exist.